### DISCUSSION TITLE
web3 modal overflowing

### DISCUSSION THREAD

**Discussion by Rickshanelorie** [WalletConnect/walletconnect-monorepo#3281] [Developer Support] [answered]
is there's a way to make the web3 modal smaller because on 100% it's too big and over flowing but on 90% it's perfect so is there's a way ??

**Comment by crypblizz8** [comment]
@glitch-txs

**Comment by glitch-txs** [comment, marked as answer]
Hi @Rickshanelorie we are aware of this, thanks for the report, the team is putting effort on web3modal v3 to have a smaller bundle for CDN
